src/polyfillstub/intersection-observer-stub.js

```diff
@@ -112,8 +112,8 @@ export class IntersectionObserverStub {
       throw new Error('root must be an Element');
     }
 
-    /** @private {?Array<!Element>} */
-    this.elements_ = [];
+    /** @private {?Set<!Element>} */
+    this.elements_ = new Set();
```

Comment:
We don't know if elements in `this.elements_` are unique ahead of time. Should I therefore keep this as an Array? @jridgewell